cli/waiter/action.py

```diff
@@ -287,3 +287,67 @@ def process_ping_request(clusters, token_name_or_service_id, is_service_id, time
                 success = True
         overall_success = overall_success and success
     return overall_success
+
+
+def kill_instance_on_cluster(cluster, instance_id, timeout_seconds):
```

Comment:
We will make this operation name more generic in subsequent commits, something like `send_signal_to_instance_on_cluster`